docs/podman-create.1.md

```diff
@@ -582,6 +582,13 @@ port to a random port on the host within an *ephemeral port range* defined by
 `/proc/sys/net/ipv4/ip_local_port_range`. To find the mapping between the host
 ports and the exposed ports, use `podman port`.
 
+**--pull**=*missing*
+
+Pull image before creating ("always"|"missing"|"never") (default "missing").
+       'missing': default value, podman will pull the image if it does not exist in the local.
```

Comment:
Suggest: "exist in the local" to "exist locally".  If you take the change, make it throughout.  That lines this up with the Buildah verbiage.